twoArrays.js

```diff
@@ -0,0 +1,33 @@
+const obj1 = [
+  {
+    first_name: "Misha",
+    second_name: "ivanushchenko",
+    dateOfBirthday: 1999,
+  },
+  {
+    first_name: "OLeg",
+    second_name: "lupa",
+    dateOfBirthday: 1974,
+  },
+  {
+    first_name: "ALex",
+    second_name: "pupa",
+    dateOfBirthday: 1997,
+  },
+  {
+    first_name: "Andrey",
+    second_name: "leps",
+    dateOfBirthday: 2005,
+  },
+  {
+    first_name: "Pop",
+    second_name: "Kek",
+    dateOfBirthday: 2018,
+  },
+];  
+
+const fixTheName = name => name = name.charAt(0).toUpperCase() + name.slice(1);
+
+const resultObj = obj1.map(item => ({fullname: [fixTheName(item.first_name), fixTheName(item.second_name)].join('_'), age: item.dateOfBirthday, ifCorresponds: item.dateOfBirthday > 2003 ? true : false}) );
```

Comment:
you can use toLowercase and next use ToUppercase for first symbols.
But thats cool too.
item.dateOfBirthday > 2003 ? true : false}) ); thats no perfect construction. You can pass only condition without ? operator

Reply:
also you can create something with `` without join.
Read more about string concatenation and ``